x-pack/plugin/rank-rrf/src/main/java/org/elasticsearch/xpack/rank/rrf/RRFRankDoc.java

```diff
@@ -42,33 +46,58 @@ public class RRFRankDoc extends RankDoc {
      */
     public final float[] scores;
 
-    public RRFRankDoc(int doc, int shardIndex, int queryCount) {
+    public final int rankConstant;
+
+    public RRFRankDoc(int doc, int shardIndex, int queryCount, int rankConstant) {
```

Comment:
I wonder if `RRFRankDoc` should be `final` to disallow inheritance?

Reply:
Yeah, makes sense! Updated.